source/index.txt

```diff
@@ -1,14 +1,34 @@
 .. _c2c-index:
 
-===============================
-MongoDB Cluster-to-Cluster Sync
-===============================
+=======================
+Cluster-to-Cluster Sync
+=======================
 
 .. default-domain:: mongodb
 
-The primary motivation of this initiative ...
+The :ref:`c2c-mongosync` tool connects MongoDB clusters so data can
+flow between them.
```

Comment:
Cluster-to-Cluster Sync provides you with continuous unidirectional data synchronization between two MongoDB clusters in the same or hybrid environments. Customers can enable Cluster-to-Cluster Sync via the mongosync utility tool.